charts/prysm/templates/service-p2p.yaml

```diff
@@ -1,5 +1,6 @@
 {{- if .Values.p2pNodePort.enabled -}}
-{{- range $i, $e := until (int $.Values.replicaCount) }}
+{{- $endpoints := uniq .Values.global.executionEndpoints -}}
```

Comment:
I think it can be confusing when you provide 3 endpoints, but only 1 consensus client is deployed. I think checking and failing is better than proceeding with fewer endpoints. Maybe we can compare array length before and after `uniq`?